solr/core/src/java/org/apache/solr/update/processor/DistributedUpdateProcessor.java

```diff
@@ -1089,94 +1057,89 @@ private boolean doVersionDelete(
       long signedVersionOnUpdate,
       boolean isReplayOrPeersync,
       boolean leaderLogic,
-      boolean forwardedFromCollection,
-      VersionBucket bucket)
+      boolean forwardedFromCollection)
```

Comment:
again, was able to balance locks and remove an indentation level.  Not clear in GitHub.